doc/tutorial/plugins/libdnf5-plugins.rst

```diff
@@ -0,0 +1,128 @@
+.. _libdnf5 plugins tutorial:
+
+LIBDNF5 Plugins
+===============
+
+These plugins enable changes to the existing dnf5 workflow.
+Modify the library's behavior at defined points, implement new logic,
+or trigger the loading of additional plugins by implementing prepared
+callbacks.
```

Comment:
What do you think of a rephrase like this?

"Possible uses of DNF5 passive plugins are modifying the libdnf5's behavior at specific breakpoints, changing or implementing the logic, or triggering the loading of additional plugins via prepared callbacks."